src/phasorpy/io.py

```diff
@@ -522,6 +540,266 @@ def read_sdt(
     return DataArray(data, **metadata)
 
 
+def read_ptu(
```

Comment:
Tested the function with a couple of files and worked as expected. The output was the same as before with using the lfdfiles.